pimpmycause/static/styles/styles.css

```diff
@@ -1 +1 @@
l-hero__cubes{margin-top:2em}}.pnl-marketing{background-color:#414;z-index:20}.pnl-marketing p{font-size:1.125rem;text-align:center}.pnl-marketing:before{border-bottom:2.375rem solid #414}.pnl-causes{background-color:#d7be01}.pnl-causes:before{border-bottom:2.375rem solid #d7be01}.pnl-marketers{background-color:#809b29}.pnl-marketers:before{border-bottom:2.375rem solid #809b29}.pnl-contact-us{background-color:#E1E0E7}.pnl-contact-us:before{border-bottom:2.375rem solid #E1E0E7}.pnl-signup{background-color:#DC0E54}.pnl-signup:before{border-bottom:2.375rem solid #DC0E54}.pnl-signup .button{background:#414}.btn-wrapper{display:flex;align-items:center;justify-content:center;height:3.25rem;margin:.5em 0;padding:0 .5em}.pnl-summary:before,.pnl-talkabout:before{width:0;height:0;border-left:2.375rem solid transparent;border-right:2.375rem solid transparent;top:-2.3125rem;left:50%;position:absolute;transform:translateX(-50%);z-index:20;content:" "}.pnl-talkabout{background-color:#414}.pnl-talkabout:before{border-bottom:2.375rem solid #414}.pnl-summary{background-color:#FEFEFE}.pnl-summary:before{border-bottom:2.375rem solid #FEFEFE}.read-more{font-weight:400}.summary-boxes li{position:relative;padding:0 .625rem;overflow:hidden;margin-bottom:.625rem}.summary-boxes li .summary-info,.summary-boxes li .summary-title{position:absolute;width:calc(100% - 1.25rem);left:.625rem;text-align:center;background-color:#f2f4f5;vertical-align:middle;text-size:1.25rem}.summary-boxes li .summary-title{bottom:-.0625rem;height:3.125rem;line-height:3.125rem;z-index:20;transition:background-color .5s,color .5s}.summary-boxes li .summary-title:before{width:0;height:0;border-left:.75rem solid transparent;border-right:.75rem solid transparent;border-bottom:.75rem solid #f2f4f5;top:-.6875rem;left:50%;transform:translateX(-50%);position:absolute;z-index:20;content:" ";transition:border-bottom-color .5s}.summary-boxes li .summary-title.click-event+.summary-info,.summary-boxes li .summary-title:hover+.summary-info{transform:translateY(0)}.summary-boxes li .summary-info{bottom:0;padding:.625rem .625rem 3.75rem;opacity:.9;z-index:10;transform:translateY(100%);transition:transform .5s}.summary-boxes li .summary-info.click-event,.summary-boxes li .summary-info:hover{transform:translateY(0)}.summary-boxes li .summary-info .read-more{font-weight:400}a,body{font-weight:100}.summary-title.click-event,.summary-title:hover{color:#f2f4f5}.page-about .pnl__title,.page-contact .pnl__title,.page-login .pnl__title,.page-register .pnl__title,.page-team .pnl__title{color:#414}.summary-pink .summary-title.click-event,.summary-pink .summary-title:hover{background-color:#DC0E54}.summary-pink .summary-title.click-event:before,.summary-pink .summary-title:hover:before{border-bottom-color:#DC0E54}.summary-purple .summary-title.click-event,.summary-purple .summary-title:hover{background-color:#414}.summary-purple .summary-title.click-event:before,.summary-purple .summary-title:hover:before{border-bottom-color
```

Comment:
The compiled css is in the repo? I imagine is for an easier deploy process, but is not a best practise.